packages/frontend/src/renderer/store/reducers.ts

```diff
@@ -13,16 +17,18 @@ import { socketReducer } from '../sagas/socket/socket.slice'
 import { modalsReducer } from '../sagas/modals/modals.slice'
 
 import appHandlers from './handlers/app'
-import waggleHandlers from './handlers/waggle'
-import contactsHandlers from './handlers/contacts'
-import channelHandlers from './handlers/channel'
-import directMessages from './handlers/directMessages'
-import directMessageChannelHandlers from './handlers/directMessageChannel'
-import mentionsHandlers from './handlers/mentions'
-import criticalErrorHandlers from './handlers/criticalError'
-import whitelistHandlers from './handlers/whitelist'
 
-const reduxStorage = createElectronStorage()
+const dataPath = process.env.APPDATA || (process.platform === 'darwin' ? process.env.HOME + '/Library/Application Support' : process.env.HOME + '/.config')
+const appPath = process.env.DATA_DIR || (process.env.NODE_ENV === 'development' ? 'Quietdev' : 'Quiet')
```

Comment:
Maybe at least make Quietdev a const that you will import here. I mean I see that you are composing similar thing here and in main.ts

Reply:
Do you think it's better to create constant for just 2 places?